src/openrct2/peep/Peep.h

```diff
@@ -529,6 +529,14 @@ enum PeepRideDecision
     PEEP_RIDE_DECISION_THINKING = 1 << 2,
 };
 
+// Flags used by PeepThoughtToActionMap
```

Comment:
This is only used in Peep.cpp so it can be moved to peep.cpp just above the `PeepThoughtToActionMap`